config/puma.rb

```diff
@@ -6,8 +6,8 @@
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
 # and maximum; this matches the default thread size of Active Record.
 #
-max_threads_count = ENV.fetch('RAILS_MAX_THREADS', 5)
-min_threads_count = ENV.fetch('RAILS_MIN_THREADS') { max_threads_count }
+max_threads_count = ENV.fetch('PUMA_MAX_THREADS', 5)
```

Comment:
Changing these variables so that their naming is clearer elsewhere (e.g., in puppet files).